### DISCUSSION TITLE
How to set video bitrate while streaming

### DISCUSSION THREAD

**Discussion by vinmediaai** [ant-media/Ant-Media-Server#5573] [FAQs]
hi guyss , 

i am new to ant media server
i doing youtube streaming for my channel . i often get error message on youtube studio stating that its receiving higher bitrate than recommend . how to set it manually 
i want to set output video quality to set to 720 p while streaming to youtube as endpoint .i am using webrtc app 

how to do it ??

**Comment by yashtandon113** [comment]
@10081994 
Thanks for asking the question. Let me try to answer.

Please see the following post for recommended YouTube upload settings for different resolutions.
https://support.google.com/youtube/answer/2853702?hl=en

If your mp4 file is encoded with very high bit rate then required for YouTube streaming then you need to re-encode your video with the recommended parameters using FFMPEG or with any other encoder like OBS etc.

If you are not familiar with FFMPEG, I recommend you to use any software encoder like OBS, VMIX etc.

**Comment by vinmediaai** [comment]
Using any other software is resource consuming and increasing my cost.

I am very new to antamedia or  to other streaming products . the default
setting of ant media is consuming 10 mbps  bandwidth, the output stream at
youtube I am receiving is 1080p . per one live stream.

i'm looking to lower my bandwidth by reducing the output quality of
the stream.so I'm looking to set 720 p , 60 FPS ,3500 KBPS bitrate ..how
can i achieve it .

but my main concern is that I want to reduce output bandwidth . I run out
of bandwidth if I stream multiple videos .

i am okay to set the setting in RTMP or webrtc .

**Reply by mekya** [reply]
Hi @10081994 ,

Yes, you're right 10Mbps per one live stream is high and as @yashtandon113 mentions, it's good to transcode the streams once and then upload transcoded VoD to Ant Media Server then stream back to youtube.

You can change the bitrate, fps, resolution or everything with FFmpeg.  This video mainly shows how to transcode the stream to a different bitrate with FFmpeg. 
https://www.youtube.com/watch?v=ULZxAghhI_I&ab_channel=LearnCodelightly 

For more information, you can proceed with FFmpeg wiki to learn more https://trac.ffmpeg.org/wiki/Encode/H.264

Let me know if we can help you in a specific way.

**Comment by vinmediaai** [comment]
Thanks, I'll check it out!
